src/thebelab.js

```diff
@@ -167,6 +167,11 @@ function renderCell(element, options) {
       .text("run")
       .click(execute)
   );
+  $cell.append(
+    $("<button class='thebelab-button thebelab-restart-button'>")
+      .text("restart")
```

Comment:
Can you add `.attr('title', 'restart the kernel')` so there's info about what it does? Maybe a similar 'run this cell' title on the run button while we're at it.

Reply:
Done.